README.md

```diff
@@ -54,7 +54,18 @@ Environment variables used to configure this service in docker are located in th
 
 ### To build the Docker container
 
-1. `DOCKER_BUILDKIT=0 docker build --build-arg SSH_PRIVATE_KEY="$(cat ~/.ssh/id_rsa)" --build-arg SSH_PRIVATE_KEY_PASSPHRASE -t 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/overseas-entities-web .`
+As of the node 18 upgrade, the app is NO longer built in the eu-west-1 region with:
+
+`DOCKER_BUILDKIT=0 docker build --build-arg SSH_PRIVATE_KEY="$(cat ~/.ssh/id_rsa)" --build-arg SSH_PRIVATE_KEY_PASSPHRASE -t 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/overseas-entities-web .`
+
```

Comment:
These line are superfluous and will become dated quite quickly. Suggest removing them and the word 'INSTEAD' on line 61

Reply:
Removed old command.